llvm/lib/Target/RISCV/RISCVInstrInfo.cpp

```diff
@@ -2937,19 +2938,47 @@ bool RISCVInstrInfo::shouldOutlineFromFunctionByDefault(
   return MF.getFunction().hasMinSize();
 }
 
+static bool IsCandidatePatchable(const MachineInstr &MI) {
```

Comment:
Function names should start with a lower case letter

Reply:
done